test_ghstack.py

```diff
@@ -97,7 +97,8 @@ def substituteRev(self, rev: str, substitute: str) -> None:
     def gh(self, msg: str = 'Update',
            update_fields: bool = False,
            short: bool = False,
-           no_skip: bool = False) -> List[Optional[ghstack.submit.DiffMeta]]:
+           no_skip: bool = False,
+           labels: List[str] = []) -> List[Optional[ghstack.submit.DiffMeta]]:
```

Comment:
danger will robinson, mutable argument default

Reply:
yeah I was wondering about that; should I just make it `Optional` and then convert `None` to `[]` when calling `submit.main`?

Reply:
also, why isn't Flake8 warning about this?